src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -184,18 +268,35 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 		}));
 	}
 
-	public sortPropertiesByImportance(props: Map<string, TablePropertiesMapEntry>): Map<string, TablePropertiesMapEntry> {
-		return new Map([...props.entries()].sort((a, b) => {
-			if (!a[1]?.displayOrder && !b[1]?.displayOrder) {
-				return 0;
-			} else if (!a[1]?.displayOrder) {
-				return -1;
-			} else if (!b[1]?.displayOrder) {
-				return 1;
-			} else {
-				return a[1].displayOrder - b[1].displayOrder;
+	public sortPropertiesByDisplayValueEquivalency(props: Map<string, TablePropertiesMapEntry>): Map<string, TablePropertiesMapEntry> {
```

Comment:
Should we do the same in other kinds of sorting as well? We can discuss this on the call as well.